site/src/api/queries/organizations.ts

```diff
@@ -156,28 +157,90 @@ export const organizationPermissions = (organizationId: string | undefined) => {
             },
             action: "create",
           },
-          viewGroups: {
-            object: {
-              resource_type: "group",
-              organization_id: organizationId,
-            },
-            action: "read",
+        },
+      }),
+  };
+};
+
+/**
+ * Fetch permissions for all provided organizations.
+ *
+ * If organizations are undefined, return a disabled query.
+ */
+export const organizationsPermissions = (
+  organizationIds: string[] | undefined,
+) => {
+  if (!organizationIds) {
+    return { enabled: false };
+  }
+
+  return {
+    queryKey: ["organizations", "permissions"],
```

Comment:
`queryKey` needs to include the `organizationIds` in it

Reply:
Ahh for some reason I thought this would get invalidated automatically if the organizations query changed, but I think actually an invalidation has to be added manually?  Would that work too or do you think adding the IDs is better?

Reply:
Actually idk what i am thinking, easy to just throw the ids on there :stuck_out_tongue: